app/assets/javascripts/discourse/templates/full-page-search.hbs

```diff
@@ -85,6 +85,7 @@
                   {{/if}}
 
                   <a class='search-link' href='{{unbound result.url}}' {{action "logClick" result.topic_id}}>
+                    {{#if result.topic.isPrivateMessage}}<span class="pm-indicator" title="{{I18n 'topic_statuses.personal_message.title'}}">{{d-icon "envelope"}}</span>{{/if}}
```

Comment:
We normally use a lowercase for `i18n` in handlebars
```suggestion
                    {{#if result.topic.isPrivateMessage}}<span class="pm-indicator" title="{{i18n 'topic_statuses.personal_message.title'}}">{{d-icon "envelope"}}</span>{{/if}}
```

Reply:
TIL if that's case insensitive 😲